huvr_client/media.py

```diff
@@ -0,0 +1,87 @@
+from typing import TYPE_CHECKING
+
+import requests
+import os
+import uuid
+
+if TYPE_CHECKING:
+    from .client import HuvrClient
+
+
+def upload_inspection_media(
+    client: "HuvrClient",
+    filepath: str,
+    project_id: int = None,
+    checklist_id: int = None,
+    checklist_line_id: str = None,
+):
+    """
+    Uploads a file to the HUVR API.
+
+    :param client: The HUVR client.
+    :param filepath: The path to the file to upload.
+    :param project_id: The project ID to associate the file with. (optional)
+    :param checklist_id: The checklist (form) ID to associate the file with. (optional)
+    :param checklist_line_id: The checklist line ID to associate the file with. (optional)
+        format: "{checklist_id}-{section_key}-{line_key}"
```

Comment:
This should not be set by client. The format is also wrong: `{checklist_id}::{section_key}::{line_key}`